tests/resources/Nimbus-Util.robot

```diff
@@ -447,6 +447,32 @@ Deploy Simple NFS Testbed
 
     [Return]  ${user}-${name}.nfs.0  ${user}-${name}.nfs.1  ${user}-${name}.esx.0  ${nfs-ip}  ${nfs-ro-ip}  ${esx-ip}
 
+Deploy Simple VC Cluster With NFS Testbed
+    [Arguments]  ${user}  ${password}  ${spec}=  ${additional-args}=
+    ${name}=  Evaluate  'NFS-' + str(random.randint(1000,9999)) + str(time.clock())  modules=random,time
+
+    Log To Console  \nDeploying Nimbus NFS testbed: ${name}
+    ${out}=  Deploy Nimbus Testbed  user=${user}  password=${password}  spec=${spec}  args=--testbedName nfs --runName ${name} ${additional-args}
```

Comment:
Danfeng has a change to define a NIMBUS_PERSONAL_USER, which is the actual nimbus account to deploy test env. NIMBUS_USER is only used to do login operation.  Please update your code to align with it. 

Please rebase your code and use jenkins job vic-pr-test to validate your change.

Reply:
I have updated code in Nimbus-Util.robot for keyword named Deploy Simple VC Cluster With NFS Testbed, it work well as expect.